src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, String name, int radius) {
+        super(color, name);
+        this.radius = radius;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (radius * radius * Math.PI);
```

Comment:
The `calculateArea` method returns an `int`, but the area of a circle is a double value due to the use of `Math.PI`. Consider changing the return type to `double` to avoid loss of precision.